test/e2e/reconciler/scheduling/controller_test.go

```diff
@@ -154,42 +153,10 @@ func TestScheduling(t *testing.T) {
 		return location.Status.AvailableInstances != nil && *location.Status.AvailableInstances == 1, fmt.Sprintf("instances in status not updated:\n%s", toYaml(location))
 	}, wait.ForeverTestTimeout, time.Millisecond*100)
 
-	binding := &apisv1alpha1.APIBinding{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "kubernetes",
-		},
-		Spec: apisv1alpha1.APIBindingSpec{
-			Reference: apisv1alpha1.ExportReference{
-				Workspace: &apisv1alpha1.WorkspaceExportReference{
-					Path:       negotiationClusterName.String(),
-					ExportName: "kubernetes",
-				},
-			},
-		},
-	}
-
-	t.Logf("Create a binding in the user workspace")
-	_, err = kcpClusterClient.ApisV1alpha1().APIBindings().Create(logicalcluster.WithCluster(ctx, userClusterName), binding, metav1.CreateOptions{})
-	require.NoError(t, err)
-
-	t.Logf("Wait for binding to be ready")
-	framework.Eventually(t, func() (bool, string) {
-		binding, err := kcpClusterClient.ApisV1alpha1().APIBindings().Get(logicalcluster.WithCluster(ctx, userClusterName), binding.Name, metav1.GetOptions{})
-		if err != nil {
-			return false, fmt.Sprintf("failed to list Locations: %v", err)
-		}
-		return conditions.IsTrue(binding, apisv1alpha1.InitialBindingCompleted), fmt.Sprintf("binding not bound: %s", toYaml(binding))
-	}, wait.ForeverTestTimeout, time.Millisecond*100)
-
-	t.Logf("Wait for placement to be ready")
-	framework.Eventually(t, func() (bool, string) {
-		placement, err := kcpClusterClient.SchedulingV1alpha1().Placements().Get(logicalcluster.WithCluster(ctx, userClusterName), "default", metav1.GetOptions{})
-		if err != nil {
-			return false, fmt.Sprintf("failed to get placement: %v", err)
-		}
-
-		return conditions.IsTrue(placement, schedulingv1alpha1.PlacementReady), fmt.Sprintf("placement is not ready: %s", toYaml(binding))
-	}, wait.ForeverTestTimeout, time.Millisecond*100)
+	t.Logf("Bind to compute workspace")
+	framework.NewWorkloadBind(t, userClusterName, source,
```

Comment:
Does the bind command wait for the placement to be ready ?

Reply:
And also the related bindings ?

Reply:
Maybe by default we want to include waiting for the placement to be ready, and created APIBindings to be bound ?
And have an option that would disable the waiting.

Reply:
add a option of timeout, similar as bind apiexport

Reply:
Yes, exactly.